CompanyEmployees.Application/Commands/CheckInAttendanceCommand.cs

```diff
@@ -0,0 +1,6 @@
+using MediatR;
+using Shared.DataTransferObjects;
+
+namespace CompanyEmployees.Application.Commands;
+
+public sealed record CheckInAttendanceCommand(AttendanceForCreationDto Attendance) : IRequest<AttendanceDto>;
```

Comment:
This public record lacks XML documentation. Since it's part of the application's public API layer (MediatR command), it should include a summary comment to describe its purpose and usage.

```suggestion
/// <summary>
/// Command to check in an employee's attendance.
/// </summary>
public sealed record CheckInAttendanceCommand(AttendanceForCreationDto attendance) : IRequest<AttendanceDto>;
```